pkg/sql/tablewriter_upsert.go

```diff
@@ -171,6 +171,13 @@ func (tu *tableUpserter) row(
 ) error {
 	tu.currentBatchSize++
 
```

Comment:
why no `oth` plumbing here for `updateConflictingRow` here?

https://github.com/jeffswenson/cockroach/blob/jeffswenson-ldr-fix-sql-take-2/pkg/sql/tablewriter_upsert.go#L297

Reply:
This code path is used by the classic sql writer. The reason it works right now is it implies there is a local version of the row, so the where clause is able to enforce LWW. The lack of cput in this code path would be a problem if we remove the origin timestamp from the batch header, which is something I plan on doing.